### ISSUE TITLE
Feedback Profile: NumericScale supertype description is wrong

### ISSUE THREAD

**Issue by arwhyte** [IMSGlobal/caliper-spec#385]
@apn114 spotted a typo: `NumericScale super type description is wrong:

### NumericScale

```
<code>NumericScale</code> inherits all properties defined by its supertype <code>NumericScale</code>
```

change to

```
<code>NumericScale</code> inherits all properties defined by its supertype <code>Scale</code>
```

**Comment by andrew-gardener** [comment]
Added fix to #383

**Comment by arwhyte** [comment]
Resolved by #383.
